SchedulerJobs/SchedulerJobs.Services/SchedulerJobs.Services.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
 	<ItemGroup>
-		<PackageReference Include="BookingsApi.Client" Version="1.25.3" />
+		<PackageReference Include="BookingsApi.Client" Version="1.27.1-beta0021" />
```

Comment:
This will need updating to not use beta once the PR merged wont it?

Reply:
We have to do it post-booking api PR approval